README.md

```diff
@@ -14,42 +14,196 @@ This project originated as a fork of Mapbox GL Native, before their switch to a
+
+    override func viewDidLoad() {
+        super.viewDidLoad()
+        mapView = MLNMapView(frame: view.bounds)
+        mapView.autoresizingMask = [.flexibleWidth, .flexibleHeight]
+        view.addSubview(mapView)
+        mapView.delegate = self
+    }
+
+    func mapView(_: MLNMapView, didFinishLoading _: MLNStyle) {
+    }
+}
+```
+
+You need to create a wrapper when using SwiftUI.
+
+```swift
+import MapLibre
+
+struct SimpleMap: UIViewRepresentable {
+    func makeUIView(context _: Context) -> MLNMapView {
+        let mapView = MLNMapView()
+        return mapView
+    }
+
+    func updateUIView(_: MLNMapView, context _: Context) {}
+}
+```
+
+You can also use [MapLibreSwiftUI](https://github.com/maplibre/swiftui-dsl), a wrapper around MapLibre Native iOS that offers a declarative API like SwiftUI.
+
+The [iOS Documentation](https://maplibre.org/maplibre-native/ios/latest/documentation/maplibre/) contains many examples and the entire API of the library. You might also want to check out the [MapLibre Native iOS `README.md`](platform/ios/README.md).
+
+## Node.js
+
+There is an [npm package](https://www.npmjs.com/package/@maplibre/maplibre-gl-native) for using MapLibre Native in a Node.js project. The source code of this project [can be found in this repository](https://github.com/maplibre/maplibre-native/tree/main/platform/node).
+
+## Qt
+
+Please check out the [`maplibre/maplibre-native-qt` repository](https://github.com/maplibre/maplibre-native-qt) to learn how to intergrate MapLibre Native with a Qt project.
+
+## Other Platforms
+
+MapLibre Native can also be built on [Linux](platform/linux/README.md), [Windows](platform/windows/README.md) and [macOS](platform/macos/README.md).
 
-MapLibre Native for iOS 6.0.0 with Metal support has been released. See the [news announcement](https://maplibre.org/news/2024-01-19-metal-support-for-maplibre-native-ios-is-here/).
- 
 ## Contributing
 
-To contribute to MapLibre Native, see [`CONTRIBUTING.md`](CONTRIBUTING.md) and (if applicable) the specific instructions for the platform you want to contribute to.
+> [!NOTE]  
+> This section is only relevant for people who want to contribute to MapLibre Native.
+
+To get started with the code base, you need to clone the the repository including all its submodules.
+
+```bash
+git clone --recurse-submodules https://github.com/maplibre/maplibre-native.git
+```
+
+- [Core C++ API Documentation](https://maplibre.org/maplibre-native/cpp/api/) (unstable).
+- [`CONTRIBUTING.md`](CONTRIBUTING.md).
+- [MapLibre Native Markdown Book](https://maplibre.org/maplibre-native/docs/book/design/ten-thousand-foot-view.html): architectural notes.
+- Everyone is free to share knowledge and information on the [wiki](https://github.com/maplibre/maplibre-native/wiki).
+
+## Android
+
+Open `platform/android` with Android Studio.
+
+More information: [`platform/android/DEVELOPING.md`](platform/android/DEVELOPING.md).
+
+## iOS
+
+You need to have 
```

Comment:
Can this line be removed or was there a specific pre-req you wanted to mention in this sentence before the "More information" link below it?

Reply:
Oops thanks.